proof/AES256TBox.cry

```diff
@@ -0,0 +1,240 @@
+// Translated from: https://github.com/LeventErkok/sbv/blob/ebf80d83d9259fa5ad42d97fdede1ae97323cfa7/Data/SBV/Examples/Crypto/AES.hs
```

Comment:
This file comes from Galois, so I guess it is a question for them!

Probably I should be importing these from wherever Galois keep them?